transform/livereloadinject/livereloadinject_test.go

```diff
@@ -31,10 +32,15 @@ func doTestLiveReloadInject(t *testing.T, bodyEndTag string) {
 	out := new(bytes.Buffer)
 	in := strings.NewReader(bodyEndTag)
 
-	tr := transform.New(New(1313))
+	u, err := url.Parse("http://localhost:1234/subpath")
+	if err != nil {
+		t.Errorf("Parsing test URL failed")
+		return
+	}
+	tr := transform.New(New(*u))
 	tr.Apply(out, in)
 
-	expected := fmt.Sprintf(`<script data-no-instant>document.write('<script src="/livereload.js?port=1313&mindelay=10&v=2"></' + 'script>')</script>%s`, bodyEndTag)
+	expected := fmt.Sprintf(`<script data-no-instant>document.write('<script src="/subpath/livereload.js?mindelay=10&v=2&port=1234&path=subpath/livereload"></' + 'script>')</script>%s`, bodyEndTag)
```

Comment:
I assume you have tested this, so I guess it's correct, but the `path=subpath/livereload` looks ... odd?

Reply:
It looks odd, but [the default][lr-1] is `path=livereload`, so with added "subpath", which is the path used in the test, it becomes `path=subpath/livereload`. Livereload [builds a full absolute URL][lr-2] from this.

 [lr-1]: https://github.com/livereload/livereload-js#options
 [lr-2]: https://github.com/livereload/livereload-js/blob/5cba9245931bbb9bbdc61703794521ce06e4402b/src/connector.js#L11-L12